executor/load_stats.go

```diff
@@ -127,9 +128,10 @@ func (e *LoadStatsInfo) Update(data []byte) error {
 			Cms:     cms,
 			Count:   tbl.Count,
 			IsIndex: 1,
-			Err:     nil}
-
-		return nil
+			Err:     nil,
+		}
+		err = statistics.SaveMetaToStorage(e.Ctx, tbl.TableID, tbl.Count, tbl.ModifyCount, tbl.Version)
```

Comment:
I think if err is nil, we should not trace it.

Reply:
If err is nil, the errors.Trace will return nil.